frontend/public/locales/en/status.json

```diff
@@ -81,5 +81,45 @@
         "application-code-valid": "The application code or client number must be valid"
       }
     }
+  },
+  "child": {
+    "page-title": "Status Checker",
+    "status-heading": "Status",
+    "empty-status": "The information you gave us doesn't match the information in our system.",
+    "check-another": "Check another status",
+    "form": {
+      "complete-fields": "Complete all fields.",
+      "application-code-label": "Application Code or Client Number",
+      "application-code-description": "Enter the number you received after applying online, or the number found in the upper right-hand corner from the letter you may have reveived from Service Canada",
+      "application-code-placeholder": "Your Application Code or Client Number",
+      "radio-legend": "Does the child have a Social Insurance Number (SIN)?",
+      "sin-label": "Social Insurance Number (SIN)",
+      "sin-description": "Enter the 9-digit SIN",
+      "option-yes": "<strong>Yes</strong>, the child has a SIN",
+      "option-no": "<strong>No</strong>, the child does not have a SIN",
+      "if-child-summary": "If the child has a single legal name",
+      "if-child-desc": "If the child goes by a single legal name, leave the first name field blank",
+      "last-name": "Last name",
+      "first-name": "First name",
+      "date-of-birth-label": "Please enter your date of birth",
```

Comment:
Based on the text provided in the Figma: 

```suggestion
      "last-name": "Child's last name",
      "first-name": "Child's first name",
      "date-of-birth-label": "Child's date of birth",
```

Screenshot of Figma:
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/c3e09303-a3e1-4c81-81f5-f8b9b153ab30)